### PR TITLE
DR-1444 Follow up: Flight map to string fix

### PR DESCRIPTION
**Pull request by snf2ye**

### CODE REVIEW

**Review by RoriCremer — APPROVED** [review]
so fast!

**Review by pshapiro4broad — APPROVED** [review]

File: src/main/java/bio/terra/stairway/FlightMap.java [hunk 1]
```diff
@@ -118,7 +118,7 @@ public String toString() {
         map.entrySet().stream()
             .map(
                 entry -> {
-                  String valString = entry.getValue().toString();
+                  String valString = String.valueOf(entry.getValue());
```
Comment (nmalfroy) [inline src/main/java/bio/terra/stairway/FlightMap.java:121]: will that return a null if entry.getValue() is null?

Comment (pshapiro4broad) [reply src/main/java/bio/terra/stairway/FlightMap.java:121]: It will return `"null"`

Comment (nmalfroy) [reply src/main/java/bio/terra/stairway/FlightMap.java:121]: Ah got it.  thanks

Comment (snf2ye) [reply src/main/java/bio/terra/stairway/FlightMap.java:121]: I don't think we actually want to add this test, but this shows the behavior:
https://github.com/DataBiosphere/jade-data-repo/compare/sh-add-string-test?expand=1

With this change, the output just add "null" as a string for the value.
{key=null}

**Pull request merged**